src/dialogs/modelBuilderImpl.ts

```diff
@@ -2,12 +2,14 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
-
```

Comment:
nit: would probably keep the blank line between header and imports for consistency with other files.